### ISSUE TITLE
Multiple input channels with different dimensions

### ISSUE THREAD

**Issue by monod91** [keras-team/keras#10227]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [X] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [X] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

-----

Hello,

I have a model receiving a 2D input `(no_samples, max_len)`, where each sample comes in the classic form `np.array([1 5 742 555 0 0])`, where each integer corresponds to the index of the given word in the input vocabulary.
This is then fed into 2 parallel Embedding layers, which output 3D embedded sequences in the form `(no_samples, max_len, emb_size)`. In particular, one Embedding layer is randomly initialized and fine-tuned during training; the other looks up into a matrix pretrained embeddings which I am not fine-tuning during training.
Then I have a LSTM and a softmax (it is a classification problem within 4 classes).
I trained the model and it converges.

I **now** want to **manually** compute also the sequences of pre-trained embedding vectors (instead of only the sequences of integers), and to feed it as an additional input to my system. I don't need to use an Embedding layer anymore, as I am not updating the second embedding matrix anyways.
(The reason for this is that I want to replace the embedding for the OOV words with a vector consisting in the average of the embeddings of the `k` surrounding words, as done in [this](http://www.aclweb.org/anthology/N16-1170) paper).

So, during preprocessing, I create the 2D input `numpy.array(no_samples, max_len)` which is then fed into the trainable Embedding layer. Then, I initialize the 3D input in the form `numpy.array(no_samples, max_len, emb_size)` to be fed directly into the LSTM layer.
I feed my two inputs as a list `[numpy.array(no_samples, max_len), numpy.array(no_samples, max_len, emb_size)]` to the `model.fit()` function.

I can compile the model.
The problem is that when I train my model in this way, the model always predicts the first class in all epochs!

Any clues why it is like that?
I checked both inputs and they seem correct to me.

Thank you in advance.
